modules/oe_content_entity/tests/src/FunctionalJavascript/CorporateEntityUiTestBase.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_entity\FunctionalJavascript;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Base class to test corporate content entity UIs.
+ */
+abstract class CorporateEntityUiTestBase extends BrowserTestBase {
+
+  /**
+   * Creates corporate entity type.
+   *
+   * @param string $entity_type_id
+   *   Entity type id.
+   * @param string $label
+   *   Entity type label.
+   * @param string $bundle
+   *   Entity type bundle.
+   */
+  protected function createCorporateEntityType(string $entity_type_id, string $label, string $bundle): void {
```

Comment:
Technically we are creating a bundle through UI.

Reply:
Function's has been updated.